<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsRelReferenceConfigDlgBase</class>
 <widget class="QWidget" name="QgsRelReferenceConfigDlgBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="mCbxShowForm">
     <property name="text">
      <string>Show embedded form</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="mComboRelation"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="mCbxAllowNull">
     <property name="text">
      <string>Allow NULL value</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="mCbxReadOnly">
     <property name="text">
      <string>Use a read-only line edit instead of a combobox</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QgsFieldExpressionWidget" name="mExpressionWidget" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Relation</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="mCbxOrderByValue">
     <property name="text">
      <string>Order by value</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="mCbxMapIdentification">
     <property name="text">
      <string>On map identification (for geometric layers only)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Display expression</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QgsCollapsibleGroupBox" name="mFilterGroupBox">
     <property name="title">
      <string>Filters</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="mAddFilterButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../images/images.qrc">
          <normaloff>:/images/themes/default/symbologyAdd.svg</normaloff>:/images/themes/default/symbologyAdd.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QToolButton" name="mRemoveFilterButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../images/images.qrc">
          <normaloff>:/images/themes/default/symbologyRemove.svg</normaloff>:/images/themes/default/symbologyRemove.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2" rowspan="4">
       <widget class="QListWidget" name="mFilterFieldsList"/>
      </item>
      <item row="1" column="0" rowspan="4">
       <widget class="QListWidget" name="mAvailableFieldsList"/>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QCheckBox" name="mCbxChainFilters">
        <property name="toolTip">
         <string>When activated, the filters will restrict the choices of fields to options that are </string>
        </property>
        <property name="text">
         <string>Chain filters</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QCheckBox" name="mCbxAllowAddFeatures">
     <property name="text">
      <string>Allow adding new features</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsFieldExpressionWidget</class>
   <extends>QWidget</extends>
   <header>qgsfieldexpressionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mComboRelation</tabstop>
  <tabstop>mCbxAllowNull</tabstop>
  <tabstop>mCbxOrderByValue</tabstop>
  <tabstop>mCbxShowForm</tabstop>
  <tabstop>mCbxMapIdentification</tabstop>
  <tabstop>mCbxReadOnly</tabstop>
  <tabstop>mFilterGroupBox</tabstop>
  <tabstop>mAvailableFieldsList</tabstop>
  <tabstop>mAddFilterButton</tabstop>
  <tabstop>mRemoveFilterButton</tabstop>
  <tabstop>mFilterFieldsList</tabstop>
  <tabstop>mCbxChainFilters</tabstop>
 </tabstops>
 <resources>
  <include location="../../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
